test/project_types/extension/features/runtimes/admin_test.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require "test_helper"
+require "project_types/extension/extension_test_helpers"
+
+module Extension
+  module Features
+    module Runtimes
+      class AdminTest < MiniTest::Test
+        include ExtensionTestHelpers::TempProjectSetup
+
+        def test_available_flags_are_supported_for_admin
+          flags = [
+            :api_key,
+            :name,
+            :port,
+            :public_url,
+            :renderer_version,
+            :shop,
+            :uuid,
+          ]
+
+          flags.each do |flag|
+            assert_equal runtime.supports?(flag), supported
+          end
+        end
+
+        def test_unsupported_flag_is_not_supported_for_admin
+          assert_equal runtime.supports?(:fake_flag), not_supported
```

Comment:
I think this should work (and will allow you to avoid defining the `not_supported` helper method)

```suggestion
          refute runtime.supports?(:fake_flag)
```

Reply:
same as above